docs/user-guide/basic-usage.md

```diff
@@ -0,0 +1,49 @@
+#  Basic usage - host name routing
+
+ingress-nginx can be used for many usecases, inside various cloud provider and supports a lot of configurations. In this section you can find a common usage scenario where a single load balancer powerd by ingress-nginx will route traffic to 2 different HTTP backend services based on the host name.
+
+First of all follow the instructions to install ingress-nginx. Then imagine that you need to expose 2 HTTP services already installed: `myServiceA`, `myServiceB`. Let's say that you want to expose the first at `myServiceA.foo.org` and the second at `myServiceB.foo.org`. One possible solution is to create two **ingress** resources:
+
+```
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ingress-myServiceA
+  annotations:
+    # use the shared ingress-nginx
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+  - host: myServiceA.foo.org
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: myServiceA
+          servicePort: 80
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ingress-myServiceB
+  annotations:
+    # use the shared ingress-nginx
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+  - host: myServiceB.foo.org
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: myServiceB
+          servicePort: 80
+```
+
```

Comment:
I think we should also note that `myServiceA`, and `myServiceB` are in the same namespace as `ingress-myServiceA` and `ingress-myServiceB` ingresses respectively.